dist/README.md

```diff
@@ -154,7 +154,35 @@ The extra variables defined in `persist/grist-env` will only be
 applied to the Grist Docker Compose service. In particular, they will
 not affect Traefik's, Dex's, or Authelia's environment.
 
-# Updating Grist
+## Advanced profile
+
+For advanced use cases where the authentication provided by Dex and
+Authelia is not adequate, it is possible to enable an advanced Docker
+Compose profile. This profile will only configure Traefik and start
+with a completely blank Grist configuration. To generate an advanced
+configuration, from a clean slate do
+
+```sh
+COMPOSE_PROFILES=advanced \
+DEFAULT_EMAIL=gristadmin@example.com \
+GRIST_DOMAIN=grist.example.com \
+./bin/bootstrap-environment
+```
+
+All of the Grist configuration must now be provided via the
+supplemental `persist/grist-env` file described in the previous
+section. At a bare minimmum, this requires setting `APP_HOME_URL`. We
```

Comment:
*minimum

Reply:
(I'm a little sad that APP_HOME_URL is still needed for no  good reason in 2025, but no action needed in this PR)

Reply:
I'll work on removing this requirement.

Reply:
Can you fix the *minimum typo?